<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomInWindow</class>
 <widget class="QWidget" name="ZoomInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_zoomInWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>165</width>
       <height>115</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #1f1f1f;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="m_pos">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: #fff;
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_color">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: #fff;
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
